src/katsdpk8spoc/workflow_controller.py

```diff
@@ -177,6 +192,7 @@ def __init__(self, namespace: int, worker_count: int = 10, ttl: int = 600):
         self.namespace = namespace
         self.name = f"product-controller-{namespace}"
         self.ttl = ttl
+        self.config = config
         self._setup_tasks(worker_count)
 
     def _setup_tasks(self, worker_count: int):
```

Comment:
As this grows maybe start splitting the node configuration from the graph configurationL

Reply:
Good point.

Reply:
This change is probably best done when we have a more accurate workflow running... I'm leaving this as is for now.